packages/messaging/android/build.gradle

```diff
@@ -0,0 +1,38 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.1'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+//    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        minSdkVersion 16
```

Comment:
https://github.com/adobe/aepsdk-messaging-android/blob/da82b1de65354013c4130d60b88bf44ebbba1c95/code/build.gradle#L39
We should use the same `minSdkVersion` number required by Android-Messaging?